Blaise.Tests.Helpers/Instrument/InstrumentHelper.cs

```diff
@@ -64,10 +64,20 @@ public void InstallInstrument(string instrumentPackage = "")
 
             if (!DoesSurveyExists(BlaiseConfigurationHelper.InstrumentName))
             {
-                _blaiseSurveyApi.InstallSurvey(BlaiseConfigurationHelper.InstrumentName,
-                    BlaiseConfigurationHelper.ServerParkName,
-                    !string.IsNullOrEmpty(instrumentPackage) ? instrumentPackage : BlaiseConfigurationHelper.InstrumentPackage,
-                    SurveyInterviewType.Cati);
+                try
+                {
+                    _blaiseSurveyApi.InstallSurvey(BlaiseConfigurationHelper.InstrumentName,
+                        BlaiseConfigurationHelper.ServerParkName, BlaiseConfigurationHelper.InstrumentPackage,
+                        SurveyInterviewType.Cati);
+                }
+                catch (Exception ex)
+                {
+                    if (ex.InnerException != null && ex.InnerException.Message.Contains("Bad Request"))
+                    {
+                        //The install failed
+                        throw new Exception($"Error trying to install questionnaire {BlaiseConfigurationHelper.InstrumentName}.  Inner Exception {ex.InnerException.Message}");
+                    }

```

Comment:
Should this re-throw the original exception if it does not match here? Currently, other exceptions will just get swallowed.